components/board/collector.go

```diff
@@ -43,7 +43,7 @@ func newAnalogCollector(resource interface{}, params data.CollectorParams) (data
 		var value int
 		if _, ok := arg[analogReaderNameKey]; !ok {
 			return nil, data.FailedToReadErr(params.ComponentName, analogs.String(),
-				errors.New("Must supply reader_name in additional_params for analog collector"))
+				errors.New("must supply reader_name in additional_params for analog collector"))
```

Comment:
@agreenb I hope this isn't checked in app in a test. let me know if so I'll add a no lint.